parity/configuration.rs

```diff
@@ -590,18 +586,6 @@ impl Configuration {
 		self.args.arg_ntp_servers.split(",").map(str::to_owned).collect()
 	}
 
-	fn dapps_config(&self) -> DappsConfiguration {
-		DappsConfiguration {
-			enabled: self.dapps_enabled(),
-			dapps_path: PathBuf::from(self.directories().dapps),
-			extra_dapps: if self.args.cmd_dapp {
```

Comment:
Do we need to remove the corresponding `CMD` and `ARG` in `parity/cli/mod.rs`? I think it's basically those:

* https://github.com/tomaka/parity/blob/8a612b66b5918e704dbb0e5484d58638867f9ae4/parity/cli/mod.rs#L29
* https://github.com/tomaka/parity/blob/8a612b66b5918e704dbb0e5484d58638867f9ae4/parity/cli/mod.rs#L542

And we probably also need to add them to `parity/deprecated.rs`.

Reply:
There's also the config serialization struct to be removed: https://github.com/tomaka/parity/blob/8a612b66b5918e704dbb0e5484d58638867f9ae4/parity/cli/mod.rs#L1223